### PR TITLE
Added everything-presence-mmwave-configurator

### PR DESCRIPTION
**Pull request by PilaScat**

### CODE REVIEW

File: templates/everything-presence-mmwave-configurator [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  <Repository>everythingsmarthome/everything-presence-mmwave-configurator</Repository>
+  <Registry>https://hub.docker.com/r/everythingsmarthome/everything-presence-mmwave-configurator/</Registry>
+  <Network>bridge</Network>
+  <Privileged>false</Privileged>
+  <Support>https://github.com/EverythingSmartHome/everything-presence-addons/issues</Support>
+  <Project>https://github.com/EverythingSmartHome/everything-presence-addons</Project>
+  <Overview>A visual configurator tool for creating zones with the Everything Presence Lite mmWave sensor.</Overview>
+  <Category>HomeAutomation: Tools:</Category>
+  <WebUI>http://[IP]:[PORT:8099]/</WebUI>
+  <Icon>https://shop.everythingsmart.io/cdn/shop/files/EST_Compact_Logo_39a737b5-afce-4e67-9a42-db994df3e7d2.png</Icon>
```
Comment (nwithan8) [inline templates/everything-presence-mmwave-configurator:13]: Download the icon and store it in this repo.

**Review by nwithan8 — APPROVED** [review]

**Comment by nwithan8** [comment]
Thanks! Been planning on getting my hands on an EP1, glad there will be the config tool on Unraid.

**Comment by PilaScat** [comment]
Yes, it is very cool, now I'll store the image on this repo

**Comment by PilaScat** [comment]
Did it, I think you need to bypass checks since I already put the final URL of the icon

**Review by nwithan8 — APPROVED** [review]

**Comment by Roxedus** [comment]
when it is not a xml file, yes that is expected

**Comment by nwithan8** [comment]
> when it is not a xml file, yes that is expected

*facepalm* My bad, I'll fix it

**Comment by PilaScat** [comment]
lol, didn't noticed

**Pull request merged**